compaction/time_window_compaction_strategy.hh

```diff
@@ -97,7 +97,7 @@ private:
         case time_window_compaction_strategy_options::timestamp_resolutions::millisecond:
             return std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::milliseconds(timestamp_from_sstable)).count();
         default:
-            throw std::runtime_error("Timestamp resolution invalid for TWCS");
+            on_internal_error(clogger, std::format("Timestamp resolution invalid for TWCS : {}", static_cast<int>(resolution)));
```

Comment:
Let's remove the `default:` case so the compiler will warn when we add new members to the enum.
Add the `on_internal_error()` below the switch, so it is invoked only when no case in the switch handles the value.

Reply:
Ack. Done.